packages/server/lib/browsers/utils.ts

```diff
@@ -1,10 +1,12 @@
 import _ from 'lodash'
 import type { FoundBrowser } from '@packages/launcher'
-// @ts-ignore
+
 import errors from '../errors'
 // @ts-ignore
 import plugins from '../plugins'
 
+const errorsChild = require('../errors-child')
```

Comment:
why did we split out this single error into `errors-child` ?

Reply:
Because the child process uses these errors and right now the child process only allows for js files. Wasn't going to tackle that as part of this PR